Resources/public/css/theme/views/languageselectionbox.css

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+.is-languageselectionbox-hidden .ez-languageselectionbox-container {
+    display: -webkit-box;
+    display: -webkit-flex;
+    display: -ms-flexbox;
+    display: flex;
+
+    background: rgba(0, 0, 0, 0);
+    -webkit-transform: translateY(-100%);
+            transform: translateY(-100%);
+}
+
+.ez-languageselectionbox-container {
+    background: rgba(0, 0, 0, 0.6);
+
+    -webkit-transform: translateY(0);
+            transform: translateY(0);
+    -webkit-transition: all 0.3s;
+            transition: all 0.3s;
+}
+
+.ez-view-languageselectionboxview {
+    background: #D8D8D8;
+    border: 1px solid #aaa;
+    border-radius: 3px;
+}
+
+.ez-view-languageselectionboxview .ez-languageselectionbox-title {
+    font-size: 130%;
+    font-weight: normal;
+    padding-bottom: 0.2em;
+    color: #666;
+}
+
+.ez-view-languageselectionboxview .ez-languageselectionbox-languages {
+    display: block;
+    background: #fff;
+    border: solid 1px #aaa;
+    border-radius: 3px;
+    width: 414px;
```

Comment:
do we need a so precise width ? At least I would express it in a relative unit like `em` (30em or something like that)
In addition, it's a theme stylesheet, this should be added in the other stylesheet.